### ISSUE TITLE
EoE25 (Eye of Eternity 25) loot table is wrong

### ISSUE THREAD

**Issue by punims** [dalaranwow/dalaran-wow#5746]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
An interesting drop dropped to day from EoE25.
Valorous Dreadnaught Legguards dropped, an issue that had been closed in the past.

**Expected behaviour**: Tell us what should happen instead
Valorous Dreadnaught Legguards should not be on the loot table.

**Steps to reproduce the problem**:

1. Kill Malygos a few times
2. Check and see if it repeats
3. Realize the bug is back

**Include proofs for this behaviour**
https://imgur.com/a/N6PWgez

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, 
Our raid ID for this week was 756, can't really show it was looted from EoE but you can ask anyone with that raid ID.

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**
